src/components/model.js

```diff
@@ -165,6 +165,20 @@ export const saveOwnerComment = async (commentText, currentComment) => {
   await updatedComment.save()
 }
 
+export const getUserComments = async (limit=10) => {
+  const comments = await UserComment.fetchList({
+    sort: '-order'
```

Comment:
The model does not have this field. The methods are not currently used and can be removed.

Reply:
done